distribution/ddf-common/src/main/resources-filtered/etc/application-definitions/security-services.json

```diff
@@ -10,15 +10,15 @@
     "mvn:ddf.security.expansion/security-expansion-metacardattr-map/${project.version}",
     "mvn:ddf.security.expansion/security-expansion-userattr-map/${project.version}",
     "mvn:ddf.security.filter/security-filter-login/${project.version}",
-    "mvn:ddf.security.idp/security-idp-client/${project.version}",
+    "mvn:ddf.security.handler/security-handler-saml/${project.version}",
     "mvn:ddf.security.interceptor/security-interceptor-guest/${project.version}",
     "mvn:ddf.security.pdp/security-pdp-authzrealm/${project.version}",
     "mvn:ddf.security.policy/security-policy-context/${project.version}",
-    "mvn:ddf.security.sts/security-sts-certificateclaimshandler/${project.version}",
-    "mvn:ddf.security.sts/security-sts-ldaplogin/${project.version}",
-    "mvn:ddf.security.sts/security-sts-ldapclaimshandler/${project.version}",
-    "mvn:ddf.security.sts/security-sts-propertyclaimshandler/${project.version}",
-    "mvn:ddf.security.sts/security-sts-realm/${project.version}",
+    "mvn:ddf.security.sts/security-claims-certificate/${project.version}",
```

Comment:
Got it. The rest of the "sts" references lying around seem to be for configurations, which I was trying not to change, since that would break compatibility.

Reply:
This one too? https://github.com/stustison/ddf/blob/stsremovedocs/platform/security/realm/security-realm-saml/src/main/java/ddf/security/realm/sts/SamlRealm.java#L14

Reply:
You only need to review the last 3 commits on this PR. However, yes the package could get changed.